pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -137,6 +142,11 @@ stdenv.mkDerivation ({
 
   postPatch = ''
     rm -rf obj-x86_64-pc-linux-gnu
+
+    # needed for enabling webrtc+pipewire
+    substituteInPlace \
```

Comment:
Should this only run if `waylandSupport`?

Reply:
That is what I added.